src/main/java/com/example/SteDziPanki/service/AirportService.java

```diff
@@ -0,0 +1,32 @@
+package com.example.SteDziPanki.service;
+
+import com.example.SteDziPanki.model.Airport;
+import com.example.SteDziPanki.repository.AirportRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class AirportService {
+
+    private final AirportRepository airportRepository;
+
+    public AirportService(AirportRepository airportRepository) {
+        this.airportRepository = airportRepository;
+    }
+
+    public List<Airport> getAirportList(){
+        return airportRepository.findAll();
+    }
+
+    public Airport getAirport(Long id){
+        return airportRepository.findById(id).orElse(null);
```

Comment:
FindById zwraca Optionala żeby pozbyć się null. Jeżeli i tak zwracasz nulla to trochę słabo.

Reply:
public Optional<Trip> getTour(Long id){
        return tripRepository.findById(id);
    }

??

Reply:
Optional<Airport> ale tak
Wtedy w endpoincie mozesz zwrócić status 404 jeżeli nie znajdzie obiektu (jest null)